tree/ntuple/v7/test/ntuple_types.cxx

```diff
@@ -210,6 +210,60 @@ TEST(RNTuple, StdTuple)
    }
 }
 
+TEST(RNTuple, StdSet)
+{
+   auto field = RField<std::set<int64_t>>("setField");
+   EXPECT_STREQ("std::set<std::int64_t>", field.GetType().c_str());
+   auto otherField = RFieldBase::Create("test", "std::set<int64_t>").Unwrap();
+   EXPECT_STREQ(field.GetType().c_str(), otherField->GetType().c_str());
+   EXPECT_EQ((sizeof(std::set<int64_t>)), field.GetValueSize());
+   EXPECT_EQ((sizeof(std::set<int64_t>)), otherField->GetValueSize());
+   EXPECT_EQ((alignof(std::set<int64_t>)), field.GetAlignment());
```

Comment:
This is not guaranteed by the implementation (which uses `alignof(std::set<std::max_align_t>)`
Maybe ` <= ` is a better check?

Reply:
:heavy_check_mark: for the type-erased check (`otherField`). Templated set fields use their actual alignment.